core/client/src/main/java/alluxio/client/file/FileInStream.java

```diff
@@ -78,18 +75,24 @@
 
   /** If the stream is closed, this can only go from false to true. */
   protected boolean mClosed;
-  /** Whether or not the current block should be cached. */
-  protected boolean mShouldCacheCurrentBlock;
-  /** Include incomplete blocks if Alluxio is configured to store blocks in Alluxio storage. */
-  private final boolean mShouldCachePartiallyReadBlock;
- /** Current position of the stream. */
+  /**
+   * Current position of the file instream. This is not always sync-ed with block instream or
```

Comment:
The comment is a bit confusing. We need to define sync clearly when making this comment. I just deleted it. 

 I was trying to comment the following two scenario:
1. mPos = the beginning of a new block because of read
2. seek(pos). After calling updateStreams, we need to do mBlockInStream.seek(pos % mBlockSize). Otherwise mPos is not sync-ed in this sense. But it is too much implementation details. I just deleted the comment